src/components/flat-table/sort/sort.component.js

```diff
@@ -16,16 +18,22 @@ const Sort = ({ children, onClick, sortType }) => {
 
   return (
     <>
+      <span hidden id={id.current}>
+        {children}, sort type {sortType || "none"}
+      </span>
```

Comment:
Currently the voice reader reads this and then reads the button `<th type='button'` below it so you have the voice reader reading the information twice

Reply:
fixed